packages/react-querybuilder/src/utils/formatQuery/formatQuery.ts

```diff
@@ -376,6 +384,7 @@ function formatQuery(ruleGroup: RuleGroupTypeAny, options: FormatQueryOptions |
           placeholderFieldName,
           placeholderOperatorName,
           validator,
+          quoteChar,
```

Comment:
This option won't be used by parameterized rule processors, so it's not necessary to pass it here.